### ISSUE TITLE
SPTAuthViewController status bar style

### ISSUE THREAD

**Issue by trenskow** [spotify/ios-sdk#383]
Now, when the `SPTAuthViewController` is an overlay, couldn't you implement so, that it takes the returns `preferredStatusBarStyle` of it's `presentingViewController` instead of just `UIStatusBarTypeDefault`?

**Comment by LandonRover** [comment]
Agreed. To get around it, just update your plist to `View controller-based status bar appearance = NO` and handle changes within `UIApplication`.

```swift
UIApplication.sharedApplication().statusBarStyle = .LightContent
```

**Comment by trenskow** [comment]
Would also work. The status bar has, though, both styles throughout my app. I have swizzled my way out of it for now.
